Source/Common/variable.env

```diff
@@ -316,6 +316,80 @@ _t4dVarConvertToCamelCase(){
 ###############################
 }
 
+_t4dVarCaseStyleConvert(){
+# Generated From $Tools4Dev_PATH/Engine/G4d/templates-project.env
+# v1.8.0
+###### _t4dVarCaseStyleConvert
+#   - Name:
+#       _t4dVarCaseStyleConvert
+###
+#   - Synopsis:
+#      Convert string to all style output
+###
+#   - Example:
+#      *  _t4dVarCaseStyleConvert -CamelCase "vincent-the-best"       = output "vincentTheBest"
+#      *  _t4dVarCaseStyleConvert -CamelCase vincent the best       = output "vincentTheBest"
+###
+#   - Options: -CamelCase -PascalCase -SnakeCase -SnakeCaseUpper -KebabCase -PascalKebabCase
+#
+###
+
+#   - Note:
+#       <Required>
+#       [Optionnal]
+#       {XXXXXXX}       = Value To Be Changed, if {URL} then replace {URL} with a valid URL
+###
+###### DOCUMENTATION BLOC #####
+########### CODE BLOC #########
+    local _t4dErrorCode=1
+
+    local _style="$1"
+    local _string="$(echo ${@:2} | sed 's|-| |g' | tr ' ' '\n')"
+    local _Res=""
+
+    case $_style in
+        -CamelCase)
```

Comment:
in order to make life of user easier, you need to made every option corresponding to outputstyle 

for example : 
camelCase
PascalCase
snake_case
Snake_Pascal_Case
kebab-case
Kebab-Pascal-Case